packages/director/src/screenshots/s3/s3.ts

```diff
@@ -1,78 +1,64 @@
-import { AssetUploadInstruction } from '@sorry-cypress/common';
-import aws from 'aws-sdk';
-import { sanitizeS3KeyPrefix } from '../utils/';
+import { fromEnv } from "@aws-sdk/credential-providers";
+import {AssetUploadInstruction} from '@sorry-cypress/common';
+import {HttpRequest} from "@aws-sdk/protocol-http";
+import {S3RequestPresigner} from "@aws-sdk/s3-request-presigner";
+import {parseUrl} from "@aws-sdk/url-parser";
+import {formatUrl} from "@aws-sdk/util-format-url";
+import {Hash} from "@aws-sdk/hash-node";
+import {sanitizeS3KeyPrefix} from '../utils/';
 import {
-  S3_ACL,
-  S3_BUCKET,
-  S3_FORCE_PATH_STYLE,
-  S3_IMAGE_KEY_PREFIX,
-  S3_READ_URL_PREFIX,
```

Comment:
@p-8-z I believe some users wants the read url to be completely different from the write S3 bucket because, probably we'd keep it?